### PR TITLE
Pirateplayer v0.5.0-2

### PR DESCRIPTION
**Pull request by mandrean**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] Commit message includes cask’s name.
- [X] `brew cask audit --download pirateplayer` is error-free.
- [X] `brew cask style --fix pirateplayer` left no offenses.
- [X] `brew cask install pirateplayer` worked successfully.
- [X] `brew cask uninstall pirateplayer` worked successfully.

### CODE REVIEW

File: Casks/pirateplayer.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask 'pirateplayer' do
+  version '0.5.0-2'
+  sha256 'e013d8e3402d12262475f7fc5cd557d5bafe9c6da92cbd2cd29ac15eaebe2e6a'
+
+  url 'http://pirateplay.se/static/pirateplayer_archive/pirateplayer-0_5_0-2.dmg'
+  appcast 'https://github.com/jackuess/pirateplayer/releases.atom',
+          checkpoint: 'f61d3c35b95a7581000877e4294b6f38fefa60a16646684f32aa08f3372ceabe'
+  name 'pirateplayer' # App's name is 'pirateplayer', not 'pirateplay'
```
Comment (vitorgalvao) [inline Casks/pirateplayer.rb]: Remove comment. It’s useless.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 75c56736474cf79006adb951b1c579e0af82610d. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by mandrean** [comment]
@vitorgalvao Notice how the url ended with "0_5_0-2.dmg", but your commit [75c5673](https://github.com/caskroom/homebrew-cask/commit/75c56736474cf79006adb951b1c579e0af82610d) changes it to #{version}.dmg and the install now fails:

**Comment by vitorgalvao** [comment]
[Easy mistake to fix](https://github.com/caskroom/homebrew-cask/commit/b6b789aee82ae65f0c553cad5b4a36e57f14100a). We could also have changed `version` to `0_5_0-2` directly.

Eschewing interpolation isn’t the solution. Making it smarter is.

**Comment by mandrean** [comment]
Awesome! Thanks for the quick response. I don't mind how you add auto-updates, as long as it works.

**Pull request closed without merging**